src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -103,9 +103,10 @@ public void equals() {
 
     @Test
     public void toStringMethod() {
-        Index targetIndex = Index.fromOneBased(1);
-        DeleteCommand deleteCommand = new DeleteCommand(targetIndex);
-        String expected = DeleteCommand.class.getCanonicalName() + "{targetIndex=" + targetIndex + "}";
+        Person personToDelete = model.getFilteredPersonList().get(1);
```

Comment:
Just wondering if generally we should use getPersonWithNric(Nric) instead of get(INDEX)